### PR TITLE
Hide Shape and Add To Shapes Lab unable to select item in group #1464

### LINKED ISSUE #1464: Hide Shape and Add To Shapes Lab unable to select item in group
These features on the context menu are unable to get the items within groups. When we try to use these features for an item in a group, we end up doing the operation on the entire group.

The expected behavior should be that the action is applied to the item in the group, similar to Edit Name

### PR DESCRIPTION
**Pull request by IanTeo**
Fixes #1464

### CODE REVIEW

File: PowerPointLabs/PowerPointLabs/ActionFramework/ShortcutsLab/Hide/HideShapeActionHandler.cs [hunk 1]
```diff
@@ -9,7 +9,13 @@ class HideShapeActionHandler : ActionHandler
     {
         protected override void ExecuteAction(string ribbonId)
```
Comment (initialshl) [inline PowerPointLabs/PowerPointLabs/ActionFramework/ShortcutsLab/Hide/HideShapeActionHandler.cs]: HideShapeHandlers are in the folder "Hide", should the folder be renamed to "HideShape"? Similarly to "TextCollection.HideSelectedShapeTag"

File: PowerPointLabs/PowerPointLabs/Models/PowerPointShapeGalleryPresentation.cs [hunk 2]
```diff
@@ -4,10 +4,11 @@
 using System.Linq;
 using System.Text.RegularExpressions;
 using System.Windows.Forms;
-using Microsoft.Office.Core;
+
 using Microsoft.Office.Interop.PowerPoint;
 using PowerPointLabs.Utils;
```
Comment (initialshl) [inline PowerPointLabs/PowerPointLabs/Models/PowerPointShapeGalleryPresentation.cs]: Can put a line break here too

**Comment by IanTeo** [comment]
What should I name the folder? I changed it to ShapesLabPane because it is the codes to open the pane from the ribbon. The other labs like resize and positions lab are placed in a folder called "Resize Lab" and "Positions lab" respectively. But I felt it might be kind of weird for the folders to be ShapesLab -> ShapesLab

**Comment by initialshl** [comment]
Maybe ShapesLab -> RibbonButton?

**Comment by IanTeo** [comment]
How about if all the handlers for the ribbon buttons are in the folder itself?

**Comment by initialshl** [comment]
Yup ok sure

**Pull request merged**